apps/docs/content/guides/auth/passwords.mdx

```diff
@@ -184,7 +184,7 @@ export async function GET(request: NextRequest) {
   redirectTo.pathname = next
 
   if (token_hash && type) {
-    const supabase = createClient()
```

Comment:
[this page](https://docs-git-chore-update-to-next-15-supabase.vercel.app/docs/guides/auth/passwords?queryGroups=flow&flow=pkce) has another `const supabase = createClient()`